crates/chainio/clients/elcontracts/src/reader.rs

```diff
@@ -547,8 +546,8 @@ impl ELChainReader {
         strategy_addr: Address,
     ) -> Result<
         (
-            IStrategyInstance<Http<Client>, SdkProvider>,
-            IERC20Instance<Http<Client>, SdkProvider>,
+            IStrategyInstance<(), SdkProvider>,
```

Comment:
What's the reason for using an empty tuple here?

Reply:
https://github.com/alloy-rs/alloy/blob/8ee24637a6f506dd66fbf2233284d767555e3524/crates/contract/src/lib.rs#L41-L48

There's no `T: Transport` on `Provider` anymore, but the sol macro wasn't updated. They made a fake trait and use `()` as a placeholder until they update, I guess.